content/lectures/lec01.md

```diff
@@ -7,91 +7,193 @@ slides: https://docs.google.com/presentation/d/1FVklEogqEGn6zsp8YOpCuynUCCvhB6mX
 draft: false
 ---
 
-# Lecture One: HTTP
+# Lecture One: Networking
 
-## Motivation
-DevOps is a philsophy, but infrastructure engineering is an important knowledge base for DevOps work. At least a cursory understanding of networking, specifically the HTTP protocol, is important for understanding the infrastructure behind the web, which is what we'll be covering for the rest of the lecture.
+## 7 Layers of the OSI Model
 
-## HTTP
+The internet is built in layers that allow us to abstract away a lot of the complexity that is inherent to networks. One common model for layers is the OSI Model (from top to bottom):
 
-HTTP is the protocol behind the web. It operates on the client/server model: the client requests something from the server, and the server responds to that request with the desired result. The server will serve many clients. HTTP underlies the orchestration that we will build up to throughout the course, and so it's important to start with foundational understanding. If we run into issues with our deployments, knowledge of how HTTP enables communication over the web is going to be vital in addressing the problem.
+- Application: End user later (HTTP, FTP, SSH, DNS)
+- Presentation: Syntax layer (SSL, SSH, IMAP, FTP, JPEG)
+- Session: Synch and send to port (APIs, sockets)
+- Transport: End-to-end connections (TCP, UDP, QUIC)
+- Network: Packets (IP, ICMP, IPSec, IGMP)
+- Data Link: Frames (Ethernet, PPP, Switch, Bridge)
+- Physical: Physical structure (Coax, Fiber, Wireless, Hubs, Repeaters)
+
+## Internet Protocol (IP)
+
+IP is the lowest level of abstraction that we will uncover in this course. You can think of an IP address as a single atomic element that lives on a network. A computer is given a IP address that consists of four octets (32 bits of data) separated by dots. A subnet is a subset of all IP addresses available. The IP ranges given for Loopback and Private IPs below represent subnets, but a subnet can be any range of IPs.
+
+There are a few special IP addresses that you might become familiar with:
```

Comment:
maybe mention subnets something like this: a few special subnets (a group of IP addresses)